components/content/ProseImg.vue

```diff
@@ -1,42 +1,60 @@
 <template>
     <span class="text-center d-block">
-        <NuxtImg :src="refinedSrc" :alt="alt" :width="width" :height="height" :class="classWithZoom" loading="lazy" format="webp" quality="80" densities="x1 x2" />
+        <NuxtImg
+            :src="refinedSrc"
+            :alt="alt"
+            :width="width"
+            :height="height"
+            :class="classWithZoom"
+            loading="lazy"
+            format="webp"
+            quality="80"
+            densities="x1 x2"
+        />
     </span>
 </template>
 
 <script setup lang="ts">
-    import { withBase } from 'ufo'
-    import { useRuntimeConfig, computed } from '#imports'
-    const props = defineProps({
-        src: {
-            type: String,
-            default: ''
-        },
-        alt: {
-            type: String,
-            default: ''
-        },
-        width: {
-            type: [String, Number],
-            default: undefined
-        },
-        height: {
-            type: [String, Number],
-            default: undefined
-        },
-        class: {
-            type: String,
-            default: ''
-        }
-    })
-    const refinedSrc = computed(() => {
-        if (props.src?.startsWith('/') && !props.src.startsWith('//')) {
-            return withBase(props.src, useRuntimeConfig().app.baseURL)
-        }
-        return props.src
-    })
+import { withBase } from 'ufo'
```

Comment:
Please fix your indent according to the team's habits :+1: 
![image](https://github.com/kestra-io/docs/assets/37618489/6c64be99-9a7f-444b-af3d-adfcfd407761)